### DISCUSSION TITLE
Can't display image inside zellij

### DISCUSSION THREAD

**Discussion by azzamsa** [wezterm/wezterm#4360] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

GNOME

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Hi.

I can use ` wezterm imgcat Untitled.png` in wezterm and it works fine, but it doesn't work inside [zellij](https://github.com/zellij-org/zellij) session.

> Note that the image protocol isn't fully handled by multiplexer sessions at this time.
>  https://wezfurlong.org/wezterm/imgcat.html

Is this expected?

Thank you for Wezterm! 🏅

**Comment by wez** [comment, marked as answer]
probably a zellij issue; does it support image protocols? You should ask the zellij folks!
